paymentsheet/src/main/java/com/stripe/android/link/LinkController.kt

```diff
@@ -349,11 +349,21 @@ class LinkController @Inject internal constructor(
          * @param email The email address to use for Link account lookup. If provided and the email
          * matches an existing Link account, the account's payment methods will be available for selection.
          * If null, the user will need to sign in or create a Link account.
-         * @param paymentMethodType Optional filter for the type of payment methods to present.
          */
         fun presentPaymentMethods(
             email: String?,
-            paymentMethodType: PaymentMethodType? = null
+        ) {
+            interactor.presentPaymentMethods(
+                launcher = coordinator.linkActivityResultLauncher,
+                email = email,
+                paymentMethodType = null,
+            )
+        }
+
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        fun presentPaymentMethodsForOnramp(
+            email: String?,
+            paymentMethodType: PaymentMethodType?
```

Comment:
👀 TBD whether we'll allow specifying `paymentMethodType` outside of onramp in the future.